plugins/scale/scale.cpp

```diff
@@ -547,16 +552,16 @@ class wayfire_scale : public wf::plugin_interface_t
 
     void process_motion(wf::point_t to)
     {
-        if (last_selected_view)
+        if (!drag_started && last_selected_view)
         {
             wf::move_drag::drag_options_t opts;
             opts.join_views = true;
             opts.enable_snap_off    = true;
             opts.snap_off_threshold = 200;
 
             drag_helper->start_drag(last_selected_view, to, opts);
-            last_selected_view = nullptr;
-        } else if (drag_helper->view)
+            drag_started = true;
+        } else if (drag_started)
```

Comment:
I think we should still check drag_helper->view here, otherwise, LGTM.

Reply:
Right, maybe we can entirely rely on it instead of the extra boolean